src/popup/popup.js

```diff
@@ -52,6 +52,31 @@ async function load() {
 
   setup_localization();
 
+  // hide action buttons
+  let { operationMenuItemsMove, operationMenuItemsCopy, operationMenuItemsGoto, operationMenuItemsTag } = await browser.storage.local.get({
+    operationMenuItemsMove: DEFAULT_PREFERENCES.operationMenuItemsMove,
+    operationMenuItemsCopy: DEFAULT_PREFERENCES.operationMenuItemsCopy,
+    operationMenuItemsGoto: DEFAULT_PREFERENCES.operationMenuItemsGoto,
+    operationMenuItemsTag: DEFAULT_PREFERENCES.operationMenuItemsTag
+  });
+
+  if (!operationMenuItemsMove) {
```

Comment:
If you are curious, what I would probably have done is to set one attribute on a parent element with the operations to show, and then use a css rule to show only those in the attribute.

This approach is totally fine though, no need to change.